public/app/plugins/datasource/postgres/query_ctrl.ts

```diff
@@ -1,12 +1,10 @@
 import _ from 'lodash';
+import appEvents from 'app/core/app_events';
```

Comment:
Some parts from this file could also probably be extract to base class/component eventually :+1: